docs/Actions.md

```diff
@@ -1,17 +1,25 @@
 ---
 layout: default
-title: "Actions"
+title: "Queryng the API"
 ---
 
-# Writing Actions
+# Querying the API
 
-Admin interfaces often have to offer custom actions, beyond the simple CRUD. For instance, in an administration for comments, an "Approve" button (allowing to update the `is_approved` property and to save the updated record in one click) - is a must have.
+Admin interfaces often have to query the API beyond CRUD requests. For instance, in an administration for comments, an "Approve" button (allowing to update the `is_approved` property and to save the updated record in one click) - is a must have.
 
-How can you add such custom actions with react-admin? The answer is twofold, and learning to do it properly will give you a better understanding of how react-admin uses Redux and redux-saga.
+How can you add such custom actions with react-admin? There are several answers to that question, and you should understand the strengths and drawbacks of each solution before choosing one.
 
-## The Simple Way
+* [Using `fetch`](#the-simple-way-using-fetch)
+* [Using the `dataProvider`](#using-the-data-provider-instead-of-fetch)
+* [Using the `withDataProvider` Decorator](#using-the-withdataprovider-decorator)
+* [Using the `<Query>` and `<Mutation>` Components](#query-and-mutation-components)
+* [Using a Custom Action Creator](#using-a-custom-action-creator)
 
-Here is an implementation of the "Approve" button that works perfectly:
+**Tip**: Most of the time, the `<Query>` and `<Mutation>` components will do everything you need wit hthe simplest API.
```

Comment:
Explain why it's the simplest API: it handle handle the data provider nicely, etc.
And add a link to the paragraph related to these components